cmd/kubectl/kubectlcobra/inventory.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+// package kubectlcobra contains cobra commands from kubectl
+package kubectlcobra
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// Inventory organizes and stores the indentifying information
+// for an object. This struct (as a string) is stored in a
+// grouping object to keep track of sets of applied objects.
+type Inventory struct {
+	Namespace string
+	Name      string
+	GVK       schema.GroupVersionKind
```

Comment:
I think this should just be GroupKind. We don't need the version to identify an object.